models/spring-ai-ollama/src/test/java/org/springframework/ai/ollama/OllamaChatModelFunctionCallingIT.java

```diff
@@ -52,23 +52,24 @@ class OllamaChatModelFunctionCallingIT extends BaseOllamaIT {
 
 	private static final Logger logger = LoggerFactory.getLogger(OllamaChatModelFunctionCallingIT.class);
 
-	private static final String MODEL = OllamaModel.LLAMA3_1.getName();
+	private static final String MODEL = "qwen2.5:3b";
```

Comment:
I got many failures with llama3.1, even after refining the prompt. With qwen2.5:3b I got all green results, and it's even a smaller model (so better for integration testing).